src/main.rs

```diff
@@ -278,15 +277,7 @@ fn main() {
 
                 // Spawning the computation of aggregated data
                 let view_client = indexer.client_actors().0;
-
-                if indexer.near_config().genesis.config.chain_id == "mainnet" {
-                    actix::spawn(
-                        circulating_supply_provider::run_circulating_supply_computation(
-                            view_client,
-                            pool,
-                        ),
-                    );
-                }
+                aggregated::spawn_aggregated_computations(view_client, pool, &indexer);
```

Comment:
Given that you get `view_client` from `indexer`, I don't see a need to pass `view_client` separately to the `spawn_aggregated_computations`